roles/bootstrap-os/tasks/bootstrap-oracle.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+- name: Check presence of fastestmirror.conf
+  stat:
+    path: /etc/yum/pluginconf.d/fastestmirror.conf
+  register: fastestmirror
+
+# fastestmirror plugin actually slows down Ansible deployments
+- name: Disable fastestmirror plugin
+  lineinfile:
+    dest: /etc/yum/pluginconf.d/fastestmirror.conf
+    regexp: "^enabled=.*"
+    line: "enabled=0"
+    state: present
+  when: fastestmirror.stat.exists
+
+- name: Get Oracle Linux Version
+  shell: "cat /etc/oracle-release | grep -oE '[0-9]+' | head -1"
+  register: version_result
+
+- name: Set Fact | Oracle Linux Version
+  set_fact:
+    oracle_version: "{{ version_result.stdout | trim}}"
+
+- name: Check EPEL Installed
+  # noqa 303 - checking for existence of package
+  shell: "rpm -qi epel-release"
+  ignore_errors: true
+  register: epel_installed
+
+- name: Get EPEL Packages
+  get_url:
+    url: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{oracle_version}}.noarch.rpm"
+    dest: "/tmp/epel-release-latest-{{oracle_version}}.noarch.rpm"
+  when: epel_installed.rc != 0
+
+- name: Install EPEL Packages
+  yum:
+    name: "/tmp/epel-release-latest-{{oracle_version}}.noarch.rpm"
+    state: present
+  environment: "{{proxy_env}}"
+  when: epel_installed.rc != 0
+
+- name: Check SELinux Installed
+  # noqa 303 - checking for existence of package
+  shell: "rpm -qi container-selinux"
+  ignore_errors: true
+  register: selinux_installed
+
+- name: Get SELinux Packages
+  get_url:
+    url: "http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.68-1.el7.noarch.rpm"
```

Comment:
This is a quite specific package. Why this one exactly?

Reply:
No reason why we need this particular rpm version. The fetch url can be parameterized if we find that valuable.